### ISSUE TITLE
Users are getting 401's when using acquireTokenSilent

### ISSUE THREAD

**Issue by ahqardah** [AzureAD/microsoft-authentication-library-for-js#1302]
>> Please follow the issue template below. Failure to do so will result in a delay in answering your question.

## Library
- [X] `msal@1.2.1`
- [ ] `@azure/msal-browser@2.x.x`
- [ ] `@azure/msal-angular@0.x.x`
- [ ] `@azure/msal-angular@1.x.x`
- [ ] `@azure/msal-angularjs@1.x.x`

## Framework
`React 16.8.6`

## Description
After upgrading from msal 1.2.0 to msal 1.2.1, users keep getting 401's if their token is expired, acquireTokenSilent isn't refreshing the token as it's supposed to. Also, noticed a bunch of `in progress` states for the login in local storage, which might be the cause of the issue (even though the user is already logged in). Below is the code we are using:
```
private _getToken(): SyncTasks.Promise<void> {
        if (this._msal.getLoginInProgress()) {
            return SyncTasks.Defer<void>().promise();
        }
        const request: Msal.AuthenticationParameters = this._formMsalAuthParams();
        return SyncTasks.fromThenable(this._msal.acquireTokenSilent(request).then(response => {
            if (this._user && this._user.accessToken !== response.accessToken) {
                // update access token info for the user.
                this._tokenReceivedCallback(response);
                this._resetBackoff();
            }
        }, error => {
            Instrumentation.log(LogTraceArea.Network, 'MSAL AcquireTokenSilent Error: ' + JSON.stringify(error));
            // Track in telemetry so we can see if problems are brewing
            if (!this._msal.getLoginInProgress()) {
                request.forceRefresh = true;
                this._msal.acquireTokenRedirect(request);
            }
        }));
    }
```

## Security
Is this issue security related?

## Regression
Did this behavior work before?

## Configuration
Please provide your MSAL configuration options.
```
const msalConfig: Msal.Configuration = {
            auth: {
                clientId: AAD_APP_ID,
                authority: 'https://login.microsoftonline.com/common',
                validateAuthority: false,
                redirectUri: window.location.origin + RouteUtils.routeForSignIn(),

                navigateToLoginRequestUrl: true,
            },
            cache: {
                cacheLocation: 'localStorage',
            },
        };
```
```js
// Provide configuration values here.
// For Azure B2C issues, please include your policies.
```

## Reproduction steps
1. 
2. 
3. 

## Browsers
Safari, Edge, Firefox, Chrome.

**Comment by Edgias** [comment]
I think this might be related to the error I am also face: **Token is not in cache for scope:openid profile user.read** then it seems to not go past this step: **Renew token for scope: openid profile user.read is in progress. Registering callback**

**Comment by AbhaysinghBhosale** [comment]
Having same issue with multiple inProgress values in localStorage.

**Comment by tnorling** [comment]
@ahqardah Is this still an issue with ```msal@1.3.0-beta.1```?

**Comment by tnorling** [comment]
This may have been fixed with `msal@1.3.0`. Please let us know if this is still an issue
